lib/include/tree_sitter/parser.h

```diff
@@ -167,22 +167,22 @@ struct TSLanguage {
 
 #define ACTIONS(id) id
 
-#define SHIFT(state_value)              \
-  {                                     \
-    {                                   \
-      .type = TSParseActionTypeShift,   \
-      .params = {.state = state_value}, \
-    }                                   \
+#define SHIFT(state_value)                \
+  {                                       \
+    {                                     \
```

Comment:
I think we should use the field name here, for consistency with the `REDUCE` macro.